Sources/SwiftKuery/Table.swift

```diff
@@ -159,16 +288,50 @@ open class Table: Buildable {
         return self
     }
     
-    /// Add a primary key to the table.
-    ///
-    /// - Parameter columns: Columns that constitute the primary key.
+    /**
+     Function to set a single `Column` instances` as a primary key, in the `Table` instance.
+     This function calls the composite primaryKey function with a single column to create a single primary key.
+     ### Usage Example: ###
+     In this example, a column for id is initialized and a `Table` instance called "personTable" is created.
+     The personTable primary key is then set to be "idColumn".
+     ```swift
+     let idColumn = Column("id", Int32.self, notNull: true)
+     public class PersonTable : Table {
+        let tableName = "personTable"
+        let id = idColumn
+        let firstName = Column("firstName", String.self, notNull: true)
+        let lastName = Column("lastName", String.self, notNull: true)
+     }
+     var personTable = PersonTable()
+     personTable = personTable.primaryKey(idColumn)
+     ```
+     */
+    /// - Parameter columns: Single Column that constitute the primary key.
     /// - Returns: A new instance of `Table`.
     public func primaryKey(_ columns: Column...) -> Self {
         return primaryKey(columns)
     }
     
-    /// Add a foreign key to the table.
-    ///
+    /**
+     Function to set multiple `Column` instances, as a composite foreign key, in the `Table` instance.
+     This function calls the composite primaryKey function with a single column to create a single primary key.
+     ### Usage Example: ###
+     In this example, columns for first and last name are initialized and a `Table` instance called personTable is created. A "personTable" foreign key is then set to be a composite of firstColumn and lastColumn, which reference columns describing the persons salary.
+     ```swift
+     let firstColumn = Column("firstName", String.self, notNull: true)
+     let lastColumn = Column("lastName", String.self, notNull: true)
+     let monthlyPay = Column("monthlyPay", Int32.self)
+     let employeeBand = Column("employeeBand", String.self)
+     public class PersonTable : Table {
+        let tableName = "personTable"
+        let firstName = firstColumn
+        let lastName = lastColumn
+        let dateOfBirth = Column("toDo_completed", String.self)
+     }
+     var personTable = PersonTable()
+     personTable = personTable.foreignKey([firstColumn, lastColumn], references: [monthlyPay, employeeBand])
+     ```
+     */
     /// - Parameter columns: An Array of columns that constitute the foreign key.
     /// - Parameter references: An Array of columns of the foreign table the foreign key references.
```

Comment:
Needs rewording to "An array of columns within the foreign table which the foreign key references."